commodore/cluster.py

```diff
@@ -94,6 +95,36 @@ def facts(self) -> Dict[str, str]:
             return {}
         return self._cluster["facts"]
 
+def load_cluster_from_file(cfg: Config, cluster_id: str) -> Cluster:
```

Comment:
It wouldn't hurt to throw an exception in this function and `local_config_query` if the provided api_url doesn't start with `file://`.